### PR TITLE
Structured data json ld

### PR DESCRIPTION
**Pull request by jaydp17**
https://developers.google.com/search/docs/guides/intro-structured-data

### CODE REVIEW

File: src/libs/json-ld.js [hunk 1]
```diff
@@ -0,0 +1,116 @@
+import I18nText from 'shared_components/I18nText';
+
+export function getServiceJsonLdData(service, canonicalUrl) {
```
Comment (fcmatteo) [inline src/libs/json-ld.js]: This function makes a bit difficult to know which fields to expect on each type of service, maybe three different functions would be better (and maybe each of them calling another with the common fields).
It's a nice to have, if you are busy just merge this.

Comment (jaydp17) [reply src/libs/json-ld.js]: Completely makes sense.
I've updated the code.

File: src/libs/json-ld.js [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+  const structuredData = {
+    '@context': 'https://schema.org/',
+    '@type': 'Hotel',
+    name: I18nText.translate(service.title),
+    image: getHeroImage(service),
+  };
+  if (service.description && I18nText.translate(service.description))
+    structuredData.description = I18nText.translate(service.description);
+  injectRatings(service, structuredData);
+  injectGeoCoordinates(service.originalLocation, structuredData);
```
Comment (jaydp17) [inline src/libs/json-ld.js]: Also, I found that we were overwriting `service.location` but we were also saving the original location in `service.originalLocation`, thus using that.

Comment (fcmatteo) [reply src/libs/json-ld.js]: Cool! Merge it whenever you want

**Pull request merged**